terraform/cp4ba/main.tf

```diff
@@ -0,0 +1,179 @@
+  triggers = { always_run = timestamp() }
+    provisioner "local-exec" {
+    command = "mkdir -p ${var.cluster_config_path}"
+  }
+}
+
+//////######################### LDAP ##################################
+module "ldap" {
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/ldap"
+//  source = "../../ldap"
+//  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/ldap"
+
+  ibmcloud_api_key      = var.ibmcloud_api_key
+  region                = var.region
+  iaas_classic_api_key  = var.iaas_classic_api_key
+  iaas_classic_username = var.iaas_classic_username
+  ssh_public_key_file   = var.ssh_public_key_file
+  ssh_private_key_file  = var.ssh_private_key_file
+  classic_datacenter    = var.classic_datacenter
+}
+
+module "cp4ba"{
+    source = "git::https://github.com/jgod1360/terraform-ibm-cloud-pak/tree/cp4ba/modules/cp4ba"
+//    source = "../.."
+    // TODO: With Terraform 0.13 replace the parameter 'enable' or the conditional expression using 'with_iaf' with 'count'
+    enable = true
+
+    openshift_version   = var.openshift_version
+    cluster_config_path = data.ibm_container_cluster_config.cluster_config.config_dir
+    cluster_name_id     = var.cluster_name_id
+//    on_vpc              = var.on_vpc
+
+    // IBM Cloud API Key
+    ibmcloud_api_key = var.ibmcloud_api_key
+
+    # Cluster
+//    on_vpc                        = var.on_vpc
+    portworx_is_ready             = var.portworx_is_ready
+    namespace                     = var.namespace
+
+    # Platform
+    Platform_Option               = var.platform_options
+    Platform_Version              = var.platform_version
+    Project_Name                  = var.project_name
+    Deployment_Type               = var.deployment_type
+
+    Username_Email                = var.entitled_registry_user_email
+    Use_Entitlement               = local.use_entitlement
+    Entitlement_Key               = file("${path.cwd}/../../entitlement.key")
+    # Registry Images
+    Local_Public_Registry_Server  = local.local_public_registry_server
+    Local_Public_Image_Registry   = local.local_public_image_registry
+    Local_Registry_Server         = local.local_registry_server
+    Local_Registry_User           = local.local_registry_user
+    Local_Registry_Password       = local.local_registry_password
+    # Storage Classes
+    Storage_Class_Name            = local.storage_class_name
+    Sc_Slow_File_Storage_Classname   = local.sc_slow_file_storage_classname
+    Sc_Medium_File_Storage_Classname = local.sc_medium_file_storage_classname
+    Sc_Fast_File_Storage_Classname   = local.sc_fast_file_storage_classname
+}
+
+////######################### DB2 ##################################
+data "external" "install_db2" {
+
+  depends_on = [
+    module.cp4ba
+  ]
+
+  program = [
+    "/bin/bash", "https://github.com/jgod1360/terraform-ibm-cloud-pak/blob/cp4ba/modules/cp4ba/scripts/install_db2_local.sh"]
```

Comment:
Obviously, this link will need to be updated to the master branch link

Reply:
Sure, I will once I merge the forked repos. Also, when I tried to merge directly on the main repos, I got a message saying that I don't have a proper access to the `ibm-hcbt:master`. I should have informed.